app/assets/stylesheets/partners.scss

```diff
@@ -4,3 +4,16 @@
   margin: 2em;
 }
 
+ul {
+  text-align: center;
+  margin: 0 0 1em 0;
+}
+
+li {
+  display: inline;
+  margin: 0 10em 0 0;
+}
+
+li:last-child {
+  margin: 0;
+}
```

Comment:
@jasonoverby So it looks like these aren't scoped narrowly enough, and so are breaking the nav at the top of the page.

<img width="1421" alt="screen shot 2016-04-05 at 10 40 12 pm" src="https://cloud.githubusercontent.com/assets/2888017/14306806/6fe572a8-fb7f-11e5-9256-9e1ce999b990.png">